src/pCN.jl

```diff
@@ -0,0 +1,18 @@
+# Preconditioned Crank–Nicolson algorithm tools
+function pCN(source::AbstractNoiseProcess{T,N,Vector{T2},inplace}, ρ;
```

Comment:
```suggestion
"""


"""
    pCN(noise::AbstractNoiseProcess, ρ; reset=true,reverse=false,indx=nothing)

Create a new, but correlated noise process from `noise` and additional entropy with correlation ρ.
```

Reply:
Should this be `pCN!`

Reply:
yes, makes sense. I'll add an extra one with `pCN` that copies the  `source`.